addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -134,10 +132,24 @@ $h6-margin-bottom: o-website-value('h6-margin-bottom') or $headings-margin-botto
 
 $headings-color: o-color('o-cc1-headings') !default;
 
-$lead-font-size: 1.125rem !default;
+$display-line-height: 1;
+
+$font-weight-bold: 600 !default;
```

Comment:
@Brieuc-brd , why?
For reference, BS default:
```
$font-weight-lighter:         lighter !default;
$font-weight-light:           300 !default;
$font-weight-normal:          400 !default;
$font-weight-bold:            700 !default;
$font-weight-bolder:          bolder !default;
```

Reply:
@anso-odoo any idea ?

Reply:
If the font weight is not recognised and is set over 500, won't it fallback to the next bigger available value? If we keep it like this, we will have a cool 600 fw when available (the most used fonts will have it) and then it will fallback to 700 if really needed. Wdyt ?

Reply:
Ok, good point 👍